voicevox_engine/model.py

```diff
@@ -58,6 +74,9 @@ class AudioQuery(BaseModel):
     outputSamplingRate: int = Field(title="音声データの出力サンプリングレート")
     outputStereo: bool = Field(title="音声データをステレオ出力するか否か")
     kana: Optional[str] = Field(title="[読み取り専用]AquesTalkライクな読み仮名。音声合成クエリとしては無視される")
+    guidedInfo: Optional[GuidedInfo] = Field(
+        title="[Readonly] Setting for guided synthesis and guided accent phrases"
+    )
```

Comment:
`guidedInfo`はguided APIでしか使われません。
なので、Queryに含めるのではなくguided APIに渡す設計のほうがわかりやすそうです。
`GuidedInfo`を返すAPIを作るというのはどうでしょう。

The `guidedInfo` is only used in the guided API.
Therefore, it would be easier to design it to be passed to the guided API instead of including it in the Query.
How about creating an API that returns `GuidedInfo`?